service/common/src/main/java/org/apache/polaris/service/auth/DefaultActiveRolesProvider.java

```diff
@@ -82,7 +82,9 @@ protected List<PrincipalRoleEntity> loadActivePrincipalRoles(
           principal.getId());
       throw new NotAuthorizedException("Unable to authenticate");
     }
-    boolean allRoles = tokenRoles.contains(BasePolarisAuthenticator.PRINCIPAL_ROLE_ALL);
+
+    // By convention, an empty set means all roles are activated
+    boolean allRoles = tokenRoles.isEmpty();
```

Comment:
As far as I understand this logic, I believe that the old code: `tokenRoles.contains(BasePolarisAuthenticator.PRINCIPAL_ROLE_ALL)` wasn't correct, because it is impossible that `tokenRoles` would contain `"PRINCIPAL_ROLE:ALL"`, since when this pseudo-role is present in the token, this translates into an empty set.